<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calibPotUI</class>
 <widget class="QWidget" name="calibPotUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0,0,0,0);
background-image:url(://Service_Calib/Service_Calib/background_pot.png);
border: 0px;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>96</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="serviceSetupDiagnosticLabel">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>10</y>
      <width>616</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bitstream Charter</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);</string>
    </property>
    <property name="text">
     <string notr="true">Service Panel/Calibrate/Lenze potentiometer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>215</y>
      <width>376</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(100, 100, 100,50);
background-image:url(://transparent.png);
background-image:url(://transparent.png);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="calibPositionLow">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>25</y>
       <width>96</width>
       <height>46</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>22</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
background-image:url(:/transparent.png);
border-image:url(:/transparent.png);

color: rgb(79, 79, 79);
border-left-color: rgb(255, 255, 153);
border-bottom-color: rgb(255, 255, 153);
font: 75 22pt &quot;DejaVu Sans&quot;;
</string>
     </property>
     <property name="text">
      <string notr="true">0</string>
     </property>
    </widget>
    <widget class="QLabel" name="valueLabel_9">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>0</y>
       <width>66</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);
background-color: rgb(255, 255, 255,0);
color: rgb(248, 248, 54);</string>
     </property>
     <property name="text">
      <string notr="true">LOW</string>
     </property>
    </widget>
    <widget class="QLabel" name="valueLabel_10">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>0</y>
       <width>66</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);
background-color: rgb(255, 255, 255,0);
color: rgb(248, 248, 54);</string>
     </property>
     <property name="text">
      <string notr="true">HIGH</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="calibPositionHigh">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>25</y>
       <width>96</width>
       <height>46</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>22</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
background-image:url(:/transparent.png);
border-image:url(:/transparent.png);

color: rgb(79, 79, 79);
border-left-color: rgb(255, 255, 153);
border-bottom-color: rgb(255, 255, 153);
font: 75 22pt &quot;DejaVu Sans&quot;;
</string>
     </property>
     <property name="text">
      <string notr="true">0</string>
     </property>
    </widget>
    <widget class="QLabel" name="valueLabel_7">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>-5</y>
       <width>146</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);
color: rgb(248, 248, 54);
background-color: rgb(255, 255, 255,0);</string>
     </property>
     <property name="text">
      <string notr="true">thresholds</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="storeButton">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>25</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(248, 248, 248,0);
background-image:url(:/transparent.png);
border-image:url(:/Pulsanti/Pulsanti/store.png);
</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>105</y>
      <width>376</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(100, 100, 100,50);
background-image:url(://transparent.png);
background-image:url(://transparent.png);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="valueLabel_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>126</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);
background-color: rgb(255, 255, 255,0);
color: rgb(248, 248, 54);</string>
     </property>
     <property name="text">
      <string notr="true">Input An1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="valueLabel_12">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>0</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);
background-color: rgb(255, 255, 255,0);
color: rgb(248, 248, 54);</string>
     </property>
     <property name="text">
      <string notr="true">Input An2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="an1Label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>111</width>
       <height>46</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string notr="true">--%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="an2Label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>111</width>
       <height>46</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string notr="true">--%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_step1">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>180</y>
      <width>161</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(://transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="helpLabel1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>105</y>
       <width>161</width>
       <height>46</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);

color: rgb(255, 118, 55);</string>
     </property>
     <property name="text">
      <string notr="true">set Pot between 
25% and 75%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame_step_1a">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(://transparent.png);
border-image:url(://Service_Calib/Service_Calib/lenze_pot_step1.png);
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_step2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>115</y>
      <width>161</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(://transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="helpLabel2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>161</width>
       <height>46</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);

color: rgb(255, 118, 55);</string>
     </property>
     <property name="text">
      <string notr="true">Move C-ARM down</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame_step_1a_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>141</width>
       <height>116</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(://transparent.png);
border-image:url(://Service_Calib/Service_Calib/lenze_pot_step2.png);
border: 0px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_step3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>240</y>
      <width>161</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(://transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="helpLabel1_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>105</y>
       <width>161</width>
       <height>46</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);

color: rgb(255, 118, 55);</string>
     </property>
     <property name="text">
      <string notr="true">set Pot to 15%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame_step_1a_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(://transparent.png);
border-image:url(://Service_Calib/Service_Calib/lenze_pot_step1.png);
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_step4">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>300</y>
      <width>161</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(://transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="helpLabel1_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>105</y>
       <width>161</width>
       <height>46</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(/transparent.png);

color: rgb(255, 118, 55);</string>
     </property>
     <property name="text">
      <string notr="true">Close the gear</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame_step_1a_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(://transparent.png);
border-image:url(://Service_Calib/Service_Calib/lenze_pot_init.png);
border: 0px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_init">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>205</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(://transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_step_1a_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(://transparent.png);
border-image:url(://Service_Calib/Service_Calib/lenze_pot_init.png);
border: 0px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_next">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>376</width>
      <height>106</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(://transparent.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="prevButton">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>5</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-image:url(:/transparent.png);
border-image:url(:/Pulsanti/Pulsanti/scroll_left.png);
</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QPushButton" name="nextButton">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>5</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-image:url(:/transparent.png);
border-image:url(:/Pulsanti/Pulsanti/scroll_right.png);
</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>375</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-image:url(:/transparent.png);
border-image:url(:/Pulsanti/Pulsanti/info_small.png);
</string>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
